src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,284 @@
+package Duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    private static ArrayList<Task> tasks = new ArrayList<Task> ();
+    private static int numberOfTasks;
+    private static boolean addTask;
+
+    private static String logo = "     ____        _        \n" +
+            "    |  _ \\ _   _| | _____ \n" +
+            "    | | | | | | | |/ / _ \\\n" +
+            "    | |_| | |_| |   <  __/\n" +
+            "    |____/ \\__,_|_|\\_\\___|\n";
+
+    private static String SEPARATION_LINE = "    ____________________________________________________________";
+
+    private static String SHORT_SPACE = "     ";
+    private static String LONG_SPACE = "       ";
+
+    private static String GREET_MESSAGE = logo + "\n" + SEPARATION_LINE + "\n" + LONG_SPACE + "Hello! I'm Duke\n" +
+            LONG_SPACE + "What can I do for you?\n" + SEPARATION_LINE + "\n";
+    private static String EMPTY_LIST_MESSAGE = SHORT_SPACE + "You do not have any task to do.";
+    private static String MARK_AS_DONE_MESSAGE = SHORT_SPACE + "Nice! I've marked this task as done:";
+    private static String ONE_TASK_IN_LIST_MESSAGE = SHORT_SPACE + "Here is the task in your list:";
+    private static String MORE_THAN_ONE_TASK_IN_LIST_MESSAGE = SHORT_SPACE + "Here are the tasks in your list:";
+    private static String WRONG_COMMAND_ERROR_MESSAGE = SEPARATION_LINE + "\n" + SHORT_SPACE +
+            "☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n" + SEPARATION_LINE + "\n";
+    private static String MISSING_TODO_CONTENT_MESSAGE = SEPARATION_LINE + "\n" +
+            SHORT_SPACE + "☹ OOPS!!! The description of a todo cannot be empty.\n" + SEPARATION_LINE + "\n";
+    private static String MISSING_DEADLINE_CONTENT_MESSAGE = SEPARATION_LINE + "\n" +
+            SHORT_SPACE + "☹ OOPS!!! The description of a deadline cannot be empty.\n" + SEPARATION_LINE + "\n";
+    private static String MISSING_EVENT_CONTENT_MESSAGE = SEPARATION_LINE + "\n" +
+            SHORT_SPACE + "☹ OOPS!!! The description of a event cannot be empty.\n" + SEPARATION_LINE + "\n";
+    private static String MISSING_DELETE_INDEX_MESSAGE = SEPARATION_LINE + "\n" +
```

Comment:
Really good usage of constants, can include `final` since they do not change

Also you may consider extracting it out to another class suc as `Messages`

Reply:
Okay! Will implement it in the next commit!